src/utils/filterContests.test.js

```diff
@@ -0,0 +1,18 @@
+import { filterContests } from './filterContests';
```

Comment:
this file should be on `test/utils/filterContests.test.js`

Reply:
We will have to refactor this kind of thing later, @yancouto set tests up like this.